app/code/Magento/Dhl/Model/Carrier.php

```diff
@@ -1154,13 +1154,16 @@ protected function _buildQuotesRequestXml()
 
         if ($this->isDutiable($rawRequest->getOrigCountryId(), $rawRequest->getDestCountryId())) {
```

Comment:
Looks like it’s better to adjust this check, or even `isDutiable` method. What do you think?